jest.config.js

```diff
@@ -2,13 +2,14 @@ module.exports = {
   automock: false,
   clearMocks: true,
   errorOnDeprecated: false,
-  moduleFileExtensions: ["js", "json"],
+  moduleFileExtensions: ["ts", "tsx", "js", "json"],
```

Comment:
Without this, Jest won't recognize `.ts` or `.tsx` files as being things you can import!

Note the order: I put `.ts` and `.tsx` first so that if Jest has to pick, it choses the TypeScript ones first. For example, if `file.js` and `file.ts` both exist in a folder (which isn't likely with this repo's config, but you never know!), we would want Jest to take the `.ts` one.

Note that this might not be true for all projects; you might end up wanting `.js` files first if some npm package gives odd issues from this.